components/infobox/wikis/leagueoflegends/infobox_patch_custom.lua

```diff
@@ -0,0 +1,66 @@
+---
+-- @Liquipedia
+-- wiki=leagueoflegends
+-- page=Module:Infobox/Patch/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local Patch = Lua.import('Module:Infobox/Patch')
+local Injector = Lua.import('Module:Widget/Injector')
+local Widgets = require('Module:Widget/All')
+
+---@class LoLPatchInfobox: PatchInfobox
+local CustomPatch = Class.new(Patch)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomPatch.run(frame)
+	local patch = CustomPatch(frame)
+	patch:setWidgetInjector(CustomInjector(patch))
+
+	return patch:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local args = self.caller.args
+	if id == 'release' then
+		return {
+			Widgets.Cell{name = 'Release Date', content = { args.release }},
+		}
+	end

```

Comment:
Not really needed right? It only changes the cell name from `Release` to `Release Date`?